### PR TITLE
add minimal example

### PR DESCRIPTION
**Pull request by Crazytieguy**
See https://github.com/trpc/trpc/pull/1948

### CODE REVIEW

File: examples/minimal/client/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@examples/minimal-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@trpc/client": "^10.0.0-alpha.45",
+    "node-fetch": "^2.6.7"
```
Comment (KATT) [inline examples/minimal/client/package.json:7]: We can get rid of this by using node 18

**Review by KATT — COMMENTED** [review]
We can make it even more minimal ☺

**Comment by Crazytieguy** [comment]
@KATT Looks like there's a failing test that's not related to my code.
Regarding dependencies - I can downgrade wait-port, but I'm not sure it's a good idea to downgrade @types/node since the example uses node 18. Is there a way to bypass this lint?

**Comment by Crazytieguy** [comment]
@KATT So should I roll back the node-18 change or is there a way to bypass the lint?

**Comment by Crazytieguy** [comment]
@sachinraja What do you think - should I roll back to node 17 or is there a way to bypass the 'common range' lint?

**Comment by sachinraja** [comment]
Ah sorry will look at this sometime today.

**Comment by sachinraja** [comment]
Let's just rollback for now @Crazytieguy

**Comment by Crazytieguy** [comment]
Well, it looks like now the common range is 18... changing back to 18

**Comment by Crazytieguy** [comment]
@sachinraja Done 😃

**Review by sachinraja — APPROVED** [review]

**Review by sachinraja — APPROVED** [review]

**Review by KATT — APPROVED** [review]

**Pull request merged**